NEMO/models.py

```diff
@@ -1043,6 +1055,18 @@ def abuse_email_list(self):
 	def reservation_email_list(self):
 		return [email for area in self.get_ancestors(ascending=True, include_self=True) for email in area.reservation_email]
 
+	@property
+	def area_calendar_color(self):
+		for a in self.get_ancestors(ascending=True, include_self=True):
+			if a._area_calendar_color:
+				return a._area_calendar_color
+		return None
```

Comment:
maybe I am missing something here, but since area_calendar_color cannot be null, when will it ever return the parent color?